inspirehep/records/indexer/tasks.py

```diff
@@ -0,0 +1,111 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+import logging
+from celery import shared_task
+from elasticsearch import NotFoundError
+from invenio_db import db
+from invenio_pidstore.models import PersistentIdentifier
+from sqlalchemy import tuple_
+from sqlalchemy.orm.exc import StaleDataError, NoResultFound
+
+from inspirehep.records.errors import MissingCitedRecordError, MissingArgumentError
+from inspirehep.records.indexer.base import InspireRecordIndexer
+
+logger = logging.getLogger(__name__)
+
+
+def index_record_task(record):
+    uuid = record.id
+
+    index_record.delay(
+        uuid=uuid,
+        record_version=record.model.version_id,
+        deleted=record.get("deleted", False),
```

Comment:
isn't this the automatically determined value if you don't pass anything for `deleted` anyway?